examples/transfer-currency.yaml

```diff
@@ -0,0 +1,72 @@
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    lets: https://lets.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/transfer-currency.yaml
+'@graph':
+
+  # accounts before
+
+  - '@id': lets:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: Alice's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: -200
+
+  - '@id': lets:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    '@type': EconomicResource
+    skos:note: Bob's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 500
+
+  - '@id': lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    '@type': Process
+    skos:note: moving credits between accounts
+
+  - '@id': lets:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: give
+    inputOf: lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    resourceInventoriedAs: lets:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    provider: https://alice.example
+    receiver: https://bob.example
+    inScopeOf: https://lets.example/
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': lets:33f11508-92b3-4095-9e03-6678cddf7adf
+    '@type': EconomicEvent
+    action: receive
+    outputOf: lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    resourceInventoriedAs: lets:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    provider: https://alice.example
+    receiver: https://bob.example
```

Comment:
What difference you would expect in application interpreting the data if only *give* had provider and receiver? So if application get your snippet here and another equivalent one but no provider or receiver on *receive*.
How do you prevent application from counting it as two different transfers on two different events, 100 LETS each?

Reply:
Why would only give have provider and receiver?

>How do you prevent application from counting it as two different transfers on two different events, 100 LETS each?

Because they are input and output of a process, each only affecting one resource.  The whole thing is a "transfer process", not just one event.  The events are giving R&R and receiving R&R.  But that duplication from 2 agents' perspectives is why I was trying to make transfer one event with 2 resources.  You wanted this I-P-O structure, and I'm trying to work with it. You got the transfer to one thing, but I though it was messy and maybe not possible in some situations to have to deduce the other side of the transfer without direct reference to it, with the transfer information attached to one event.